openkfw/TruBudget
# 1575: Document telemetry and add separate tracer for RpcClient

### Checklist




### Description


### How to test

1. Ask me for Application Insights connection string (or get it from Azure portal)
2. Set environment variables (see docs)
3. Do some stuff
4. Check the telemetry data




Closes #1575
